integration_tests/test_model_upload.py

```diff
@@ -46,19 +45,6 @@ def wrapper(*args: object, **kwargs: object) -> ReturnType:
     return decorator
 
 
-@retry(times=5, delay_seconds=10, exception_to_check=BackendError)
```

Comment:
No need to retry anymore because we support creating a new version even if the current version is not set (no versions for the instance is in READY state).